test/e2e/simple_e2e_test.go

```diff
@@ -0,0 +1,29 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+)
+
+// This is actually a unit test, but it does exercise the necessary helper functions.
+
+func TestE2ESucceeds(t *testing.T) {
```

Comment:
What is the point of these? We should drop them from this repo. They should not be in the hack repo.

Reply:
This is for the smoke e2e tests to make sure the base e2e test flow is not broken.